server-nodejs/extensions/jwt.js

```diff
@@ -28,6 +28,17 @@ module.exports = function (app) {
         } else {
            obj = {'email' : req.body.username};
         }
+        let jwtRegex = RegExp('^[0-9]*[a-z]\$');
+        let jwtUnlimited = false;
+        let jwtExp = { expiresIn: conf.exp*60 };
+        if ("undefined" != req.body.jwtTime) {
+            if ("true" === req.body.jwtTime) {
+                jwtUnlimited = true;
+            } else if (jwtRegex.test(req.body.jwtTime) && !nameRegex.test(req.body.jwtTime)) {
+                jwtUnlimited = req.body.jwtTime;
+                jwtExp = { expiresIn: req.body.jwtTime };
+            }
+        }
```

Comment:
To set token lifespan, the user must inform a lifetime like "1d" or "1h". That's why, the test with regular expression stop all other string.